### ISSUE TITLE
Svelte EachBlock Node is broken with key `-`.

### ISSUE THREAD

**Issue by shamokit** [markuplint/markuplint#1286]
- Markuplint version: `3.15.0`
- @markuplint/svelte-parser: `3.12.0`
- Node.js version: `20.10.0`
- OS: macOS

## Describe the bug
error message
`error: The EachBlock is invalid element (xx:xx): Broke mapping nodes. (parse-error)`

### OK
```html
{#each list as item, i (i)}
...
{/each}
```
```html
{#each list as item, i (`${i}`)}
...
{/each}
```
```html
{#each list as item, i (`${i}x${i}`)}
...
{/each}
```

### NG

```html
{#each list as item, i `${i}-`)}
...
{/each}
```
```html
{#each list as item, i `${i}-${i}`)}
...
{/each}
```

The hyphen is suspicious.

**Comment by YusukeHirao** [comment]
@shamokit 

I have investigated the issue, and the problem appears in the syntax itself. There is a missing opening parenthesis. This would result in a parse error in Svelte itself as well. The minus sign is not the issue. Please check this in the Svelte playground for confirmation.
https://svelte.dev/repl/8773eb060f9e46d98a47584d5ea00299?version=4.2.8

**Comment by shamokit** [comment]
@YusukeHirao Sorry. That's a typo…

You can confirm that the hyphen causes an error in this repository.
https://github.com/shamokit/markuplint-1286

**Comment by YusukeHirao** [comment]
@shamokit 

It had been resolved in `v4.0.0-alpha.8`.
Please check it.

**Comment by shamokit** [comment]
@YusukeHirao I have confirmed the fix🎉 thank you!
